EXPO/screens/HomeScreen.js

```diff
@@ -1,188 +1,676 @@
-import React from 'react';
+/*import React from "react";
```

Comment:
clean the commented code

Reply:
Changes has been done as per suggestions.